airgun/entities/contentview_new.py

```diff
@@ -23,25 +24,91 @@ def search(self, value):
         view = self.navigate_to(self, 'All')
         return view.search(value)
 
+    def publish(self, entity_name, values=None):
+        """Publishes to create new version of CV and promotes the contents to
+        'Library' environment.
+        :return: dict with new content view version table row; contains keys
+            like 'Version', 'Status', 'Environments' etc.
+        """
+        view = self.navigate_to(self, 'Publish', entity_name=entity_name)
+        if values:
+            view.fill(values)
+        view.next.click()
+        view.finish.click()
+        view.progressbar.wait_for_result()
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        return view.versions.table.read()
+
+    def check_if_blank_in_french(self):
```

Comment:
I would rather name this differently `read_french_lang()`, here you are not checking this could be part of the test.

Reply:
Yes. This is just a read in french without any assertion.

Reply:
ah yeah, that does make more sense. good suggestion!

Reply:
@sambible Good to add doc string which would describe functionality